index.rst

```diff
@@ -8,153 +8,105 @@
-configured.
-
-Moreover we must be able to, at any point, identify what set of configurations
-a component is using, what sets are available and to be able to transition
-between different sets. During commissioning and engineering runs, users will
-also be interested in building, validating and verifying new sets of
-configurations.
-
-The process of building new sets of configurations will vary considerably for
-each component. In some cases, the configuration may be a simple set of
-host names and ports that the component connects to. In other cases the
-configuration may require the calculation of complex lookup tables (e.g. M1M3
-and M2) or model coefficients (e.g. pointing component), probably requiring
-on-sky time for acquiring the data, and complex software to analyse and derive
-the configuration products.
-
-Verification of a new configuration, in this context, mainly involves the
-process of guaranteeing that the configuration has the correct schema; the
-input values have the correct types and respect any specified range. Validating
-that a configuration is good for operation is a much more involved procedure
-and may require on-sky time.
-
-In this tech-note we specify how components deal with these different aspects
-of configuration and the basics procedures required to build and update them.
+The Vera Rubin Observatory control system is highly distributed, containing multiple components that must operate in consonant.
+In order to achieve reliable operation it is fundamental to guarantee that all components are properly configured.
+
+Moreover we must be able to, at any point, identify what set of configurations a component is using, what sets are available and to be able to transition between different sets.
+During commissioning and engineering runs, users will also be interested in building, validating and verifying new sets of configurations.
+
+The process of building new sets of configurations will vary considerably for each component.
+In some cases, the configuration may be a simple set of host names and ports that the component connects to.
+In other cases the configuration may require the calculation of complex lookup tables (e.g. M1M3 and M2) or model coefficients (e.g. pointing component), probably requiring on-sky time for acquiring the data, and complex software to analyze and derive the configuration products.
+
+Verification of a new configuration, in this context, mainly involves the process of guaranteeing that the configuration has the correct schema; the input values have the correct types and respect any specified range.
+Validating that a configuration is good for operation is a much more involved procedure and may require on-sky time.
+
+In this tech-note we specify how components deal with these different aspects of configuration and the basics procedures required to build and update them.
 
 .. _section-system-requirements:
 
 System Requirements
```

Comment:
I get why this is here... but 98% of people reading this won't care about the requirements. Maybe put this in an appendix and then just put a sentence in the intro saying the requirements driving the design are in appendix X.